lib/terraforming/resource/iam_group.rb

```diff
@@ -16,23 +16,23 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_group")
+        apply_template(@client, 'tf/iam_group')
       end
 
       def tfstate
         iam_groups.inject({}) do |resources, group|
           attributes = {
-            "arn"=> group.arn,
-            "id" => group.group_name,
-            "name" => group.group_name,
-            "path" => group.path,
-            "unique_id" => group.group_id,
+            'arn'=> group.arn,
```

Comment:
__[rubocop]__
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)